### PR TITLE
Use the nightly release for the cvp-common v4.8 job

### PR DESCRIPTION
**Pull request by dirgim**
Temporary change to ensure that rehearsal jobs don't break

### CODE REVIEW

File: ci-operator/config/redhat-operator-ecosystem/cvp/redhat-operator-ecosystem-cvp-ocp-4.8.yaml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 releases:
   latest:
-    release:
-      channel: stable
+    candidate:
+      architecture: amd64
+      product: ocp
+      relative: 1
```
Comment (petr-muller) [inline ci-operator/config/redhat-operator-ecosystem/cvp/redhat-operator-ecosystem-cvp-ocp-4.8.yaml:6]: No need for `relative` afaik

**Comment by petr-muller** [comment]
@dirgim needs `make jobs` then LGTM

**Review by petr-muller — APPROVED** [review]
> @dirgim: you cannot LGTM your own PR.

Nice try, @dirgim  :trollface:

**Comment by dirgim** [comment]
> > @dirgim: you cannot LGTM your own PR.
> 
> Nice try, @dirgim :trollface:

Well, it looked good to me :D

**Pull request merged**